xee/ext.py

```diff
@@ -448,8 +451,8 @@ def project(self, bbox: types.BBox) -> types.Grid:
             # Since the origin is in the top left corner, we want to translate
             # the start of the grid to the positive direction for X and the
             # negative direction for Y.
-            'translateX': x_origin + self.scale_x * x_start,
```

Comment:
You still need the offset within the bounding box (i.e. `translateX + self.scale_x * x_start`). Otherwise, `translateX` is just the origin of the image (not the region to be fetched). It's possible `x_start` and `y_start` will need to be adjusted based on the sign of `scale_x` and `scale_y` too.

As it's written, I think this code would only work for images that can be fetched using a single tile.

Reply:
Thank you for the suggestions. I updated the code.